python/lsst/faro/measurement/ForcedSourceTableMeasurement.py

```diff
@@ -80,12 +74,18 @@ class ForcedSourceTableMeasurementTask(CatalogMeasurementBaseTask):
 
     def runQuantum(self, butlerQC, inputRefs, outputRefs):
         inputs = butlerQC.get(inputRefs)
-        kwargs = {"band": butlerQC.quantum.dataId['band']}
+        kwargs = {"currentBands": butlerQC.quantum.dataId['band']}
+
+        columns = self.config.measure.columns()
+        for column in self.config.measure.columnsBand():
+            columns.append(kwargs["currentBands"] + "_" + column)
+        
+        columnsWithSelectors = self._getTableColumns(columns, kwargs["currentBands"])
+        catalog = inputs["catalog"].get(parameters={"columns": columnsWithSelectors})
 
-        columns = self.config.columns.list()
         tmp_catalog = inputs["catalog"].get(parameters={"columns": columns})
         # Extract only the entries from the band of interest:
-        kwargs["catalog"] = tmp_catalog[tmp_catalog.band == kwargs["band"]]
+        kwargs["catalog"] = tmp_catalog[tmp_catalog.band == kwargs["currentBands"]]
```

Comment:
Should this be `tmp_catalog.band.isin(kwargs["currentBands"])` instead of `==`? (I'm not sure that this would work if there's more than one entry in currentBands.)

Reply:
I am calling this current bands so that the measurement run method has a consistent keyword. But in the case of:

- VisitTableMeasurementTask and DetectorTableMeasurementTask `currentBands` will always be none
- TractTableMeasurementTask ForcedSourceTableMeasurementTask and PatchTableMeasurementTask `currentBands` will always be a single band 
- TractMultiBandTableMeasurementTask ForcedSourceMultiBandTableMeasurementTask and PatchMultiBandTableMeasurementTask `currentBands` will always be a set of bands currently specified in the *TableMeasurementTask config 

Open to changes on how to implement this if the current method seems confusing. 


Also for the MultiBandTableMeasurementTask should the band specification be moved to the task config?
 i.e. Line 144 of ForcedSourceTableMeasurement goes from    
`kwargs= {"currentBands": self.config.bands.list()})`
To 
`kwargs= {"currentBands": self.measure.config.bands.list()})`